components/bms_worker/include/BatteryMonitoring.h

```diff
@@ -59,9 +59,9 @@ typedef enum
 
 typedef struct
 {
-    uint16_t   voltage;
-    uint16_t   parasitic_corr;
-    uint16_t   relative_soc;
+    float32_t  voltage;
```

Comment:
if you're storing these in an array, be weary of the fact that you've just doubled your flash and/or ram usage significantly by doing this (6 bytes each vs. 12 bytes each)